modules/nested-elements/assets/js/editor/nested-repeater/hooks/data/document/elements/create/nested-repeater-adjust-container-titles.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * On each nested widget creation.
+ *
+ * For each nested repeater container, set the container title,
+ * according to the nested repeater title, e.g: the result will be 'Tab #1', 'Tab #2' and so on instead of 'Container'.
+ * '_title' is used by the navigator.
+ */
+export class NestedRepeaterAdjustContainerTitles extends ( $e.modules.hookData.After ) {
+	getId() {
+		return 'nested-repeater-adjust-container-titles';
+	}
+
+	getCommand() {
+		return 'document/elements/create';
+	}
+
+	getConditions( args ) {
+		const { model } = args;
+
```

Comment:
Please note that `move` and `duplicate` commands also runs the `create` command.  Please validate they don't change the title.
(try the `drop` command or check it's the first in trace)

Reply:
Tested, Everything works fine there is no need to change the code.